flake.nix

```diff
@@ -41,7 +41,7 @@
       ];
       perSystem = { config, self', inputs', system, pkgs, lib, ... }:
         let
-          rustToolchain = inputs'.fenix.packages.stable.toolchain;
+          rustToolchain = inputs'.fenix.packages.complete.toolchain;
```

Comment:
```suggestion
          rustToolchain = inputs'.fenix.packages.latest.toolchain;
```

Reply:
`latest` is less stable than `complete`, which is also a nightly toolchain. I would use `stable`.

Reply:
@marijanp Please take a look at this, I remember that we were having issues with `complete` ?

Reply:
That was due to a version mismatch. What we did is merge a `latest` rust-std for wasm with a `stable` toolchain. I reverted this to use stable. See the passing pipeline.